src/components/Panels/DeployContract.vue

```diff
@@ -417,9 +450,15 @@ export default {
           // eslint-disable-next-line no-empty
         } catch (e) {}
 
+        const validatedType = getParamType({ type: item.type });
+
         return {
           vname: item.vname,
-          type: item.type,
+          type:
+            validatedType.parsed !== validatedType.initial &&
+            validatedType.parsed === "ByStr20"
+              ? validatedType.parsed
+              : item.type,
```

Comment:
`getParamType` can return `string` if valid and `undefined` if invalid. If undefined is returned, then you can use the original input value.